.github/workflows/test_notebooks.yml

```diff
@@ -5,15 +5,19 @@ on:
   push:
     branches: [ develop, stable, nbtests ]
     paths-ignore:
-      - '**/*.md' # ignore markdown files
-      - '**/*.rst' # ignore restructured text files
+      - 'DEA_Sandbox.ipynb' # ignore landing page
+      - 'DEA_notebooks_template.ipynb' # ignore template
+      - '*.md' # ignore all markdown files
+      - '*.rst' # ignore all restructured text files
       - '.github/**' # ignore anything in .github folder
       - '!.github/workflows/test_notebooks.yml' # except test_notebooks.yml
   pull_request:
     branches: [ develop, stable ]
     paths-ignore:
-      - '**/*.md'
-      - '**/*.rst'
+      - 'DEA_Sandbox.ipynb'
+      - 'DEA_notebooks_template.ipynb'
+      - '*.md'
+      - '*.rst'
       - '.github/**'
       - '!.github/workflows/test_notebooks.yml'
```

Comment:
Maybe we should ignore everything in the root directory except for Notebook files:

```yaml
      - '*'
      - '!*.ipynb'
```

Reply:
I'd be happy to ignore all top-level files - the two notebooks there are either entirely Markdown or are not intended to be run by our users (both aren't included in our test coverage anyway, and `DEA_notebooks_template.ipynb` [isn't even synced to the Sandbox](https://github.com/GeoscienceAustralia/dea-sandbox/blob/develop/docker/assets/sync_repo#L15)).

Reply:
I agree. Sounds good.